docs/data/base/components/select/select.md

```diff
@@ -93,10 +94,58 @@ For the sake of brevity, the rest of the demos throughout this doc will not use
 ### Multi-select
 
 The Select component lets your users select multiple options from the list.
+In contrast to the single-selection mode, the options popup doesn't close after an item is selected, enabling users to continue choosing more options.
+
 Set the `multiple` prop to turn on the multi-selection mode.
 
 {{"demo": "UnstyledSelectMultiple.js", "defaultCodeOpen": false}}
 
+Note that in the multiple selection mode, the `value` prop (and `defaultValue`) is an array.
+
+### Controlled select
+
+Select can be used as an uncontrolled or controlled component.
+
+:::info
+
+- The value is **controlled** when its parent manages it by providing a `value` prop.
+- The value is **uncontrolled** when it is managed by the component's own internal state. This state can be initialized using the `defaultValue` prop.
+
+Learn more about the _Controlled and uncontrolled_ pattern in the [React documentation](https://react.dev/learn/sharing-state-between-components#controlled-and-uncontrolled-components).
```

Comment:
Very tiny style adjustment for the sake of uniformity with other Core docs with this callout.

```suggestion
Learn more about controlled and uncontrolled components in the [React documentation](https://react.dev/learn/sharing-state-between-components#controlled-and-uncontrolled-components).
```